samples/Azure.Management.Storage/Generated/LongRunningOperation/BlobRestoreStatusRestoreBlobRangesOperation.cs

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.Management.Storage.Models;
+using Azure.ResourceManager;
+
+namespace Azure.Management.Storage
+{
+    public class BlobRestoreStatusRestoreBlobRangesOperation : ArmOperation<BlobRestoreStatus>
```

Comment:
name should be [Resource][Method]Operation

Reply:
fixed